jupyter_smart_on_fhir/server_extension.py

```diff
@@ -7,23 +8,47 @@
 import hashlib
 import base64
 from jupyter_smart_on_fhir.auth import SMARTConfig, generate_state
+from traitlets import List, Unicode
 
-smart_path = "/extension/smart"
-login_path = "/extension/smart/login"
-callback_path = "/extension/smart/oauth_callback"
+smart_path = "/smart"
+login_path = "/smart/login"
+callback_path = "/smart/oauth_callback"
 
 
 def _jupyter_server_extension_points():
-    return [{"module": "jupyter_smart_on_fhir.server_extension"}]
+    return [
+        {"module": "jupyter_smart_on_fhir.server_extension", "app": SmartExtensionApp}
+    ]
 
 
-def _load_jupyter_server_extension(serverapp: ServerApp):
-    handlers = [
-        (smart_path, SmartAuthHandler),
-        (login_path, SmartLoginHandler),
-        (callback_path, SmartCallbackHandler),
-    ]
-    serverapp.web_app.add_handlers(".*$", handlers)
+class SmartExtensionApp(ExtensionApp):
+    name = "fhir"
+    default_url = "/smart"
+    load_other_extensions = True
+    file_url_prefix = "/fhir"
```

Comment:
I'm not sure we want the options other than `name` defined here.

```suggestion
    name = "smart_on_fhir"
```

- default_url is the default landing page for users - the `/smart` page doesn't work except as a redirect target, so is probably not appropriate here
- `load_other_extensions` is the default already
- `file_url_prefix` is for when launching files for the command-line, and this extension shouldn't change that and doesn't register a `/fhir` handler, as far as I can tell.

Reply:
Thanks; [this tutorial[(https://jupyter-server.readthedocs.io/en/latest/developers/extensions.html) lists them as required traits but I guess they have sensible default values already. I'll take them out